### PR TITLE
chore(deps): :arrow_up: bump @yi-xu-0100/conventional-commit-types-i18n from 1.4.0 to 1.5.0

### PR DESCRIPTION
**Pull request by yi-Xu-0100**
This version allows the package to bundle the JSON files.
It will lead to deleting the useless copy-webpack-plugin.

The `eslint` will throw the error: 

```error
error TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 
'{ feat: { description: string; title: string; }; fix: { description: string; title: string; };
docs: { description: string; title: string; };style: { description: string; title: string; };
refactor: { description: string; title: string; }; ... 5 more ...; revert: { ...; }; }'.
      No index signature with a parameter of type 'string' was found on type
'{ feat: { description: string; title: string; }; fix: { description: string; title: string; };
docs: { description: string; title: string; }; style: { description: string; title: string; };
refactor: { description: string; title: string; }; ... 5 more ...; revert: { ...; }; }'.
```

So I add `ts-ignore`. 😥 but I can promise that it will do well in the code.

If you can find a way to fix it, I will try to adjust my code. 😅

### CODE REVIEW

File: src/lib/prompts.ts [hunk 1]
```diff
@@ -24,6 +24,13 @@ import commitMessage, {
 import commitlint from './commitlint';
 import localize, { locale } from './localize';
 
+type ConventionalCommitsTypes = {
```
Comment (vivaxy) [inline src/lib/prompts.ts:27]: Add type definition, and remove `ts-ignore`

**Review by vivaxy — APPROVED** [review]

**Comment by vivaxy** [comment]
@yi-Xu-0100

I've updated the type definition. Feel free to merge. 😀

File: src/lib/prompts.ts (hunk 1 above)
Comment (yi-Xu-0100) [reply src/lib/prompts.ts:27]: Thanks! Should I update the definition into my package?

Comment (vivaxy) [reply src/lib/prompts.ts:27]: @yi-Xu-0100 Your package is fine. Types definitions in your package are more strict than this extension. 😀

I think we should work on the types definitions within the extension. But I have no better ideas right now.

**Pull request merged**